.storybook/stories/datagrid/datagrid-row.stories.ts

```diff
@@ -10,6 +10,49 @@ import { moduleMetadata, StoryFn, StoryObj } from '@storybook/angular';
 
 import { elements } from '../../helpers/elements.data';
 
+export default {
```

Comment:
Moving large chunks of code in the middle of an already-large PR makes things harder to review. Please either put these back or move them in a separate PR where that is the only change.

Reply:
Reverted this change.